### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by dimqua** [omarroth/invidious#180]
Is it possible to add a support of translations of the UI?

**Comment by omarroth** [comment]
Absolutely! Text would need to be pulled out into a `locales/en-US.json` similar to [FreeTube locales](https://github.com/FreeTubeApp/FreeTube/blob/development/locales). I haven't had a chance to do just yet.

**Comment by omarroth** [comment]
Added `locales/en-US.json` with d211d8f.

**Comment by omarroth** [comment]
I agree. I submitted a request to Weblate, which may take a while to be approved.

**Comment by dimqua** [comment]
Is it possible to automatically import a translations from Weblate time to time?

**Comment by omarroth** [comment]
Weblate produces a lot of commits for each translation that make it difficult to read the log, and doesn't have any tooling for squashing automatically, so I'm planning on doing that work manually. I do expect to sync it often.
